modules/serverless-list-broker-kn.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-list-broker-kn_{context}"]
+= List existing brokers using the kn CLI
+
+.Prerequisites
+* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on your {product-title} cluster.
+* You have installed the `kn` CLI.
+
+.Procedure
+. List all existing brokers:
++
+
+[source,terminal]
+----
+$ kn broker list
+----
```

Comment:
@rhuss and last one here.... :smile: if not I can start a cluster tomorrow at some point to get these